packages/@react-spectrum/searchfield/src/SearchField.tsx

```diff
@@ -23,6 +23,7 @@ import {useSearchField} from '@react-aria/searchfield';
 import {useSearchFieldState} from '@react-stately/searchfield';
 
 function SearchField(props: SpectrumSearchFieldProps, ref: RefObject<TextFieldRef>) {
+  props = useSlotProps(props, 'searchfield');
```

Comment:
I think we may need to allow for some extra props to be passed in on context and context only for this kind of composition
specifically we need to allow for an `inputClassName` so that you can merge it down below in `inputClassName={classNames(styles, 'spectrum-Search-input', slot_inputClassName)}`

Reply:
@snowystinger If you don't mind, can you take a look at [bbc69ac](https://github.com/adobe/react-spectrum/pull/2062/commits/bbc69accc9ff0ca3423056550243f363ca8a8d8c) to see if that's what you were thinking for this?